pkg/api/errors/metdata.go

```diff
@@ -0,0 +1,34 @@
+package errors
+
+import (
+	"fmt"
+    "net/http"
+	"google.golang.org/grpc/codes"
+	kiterrors "github.com/dapr/kit/errors"
+)
+
+func IncomingContextMetadataNotFound() error {
```

Comment:
We have a generic not found error in the errors.go file in this same directory you can use instead of creating one specific to the Metadata API